package.json

```diff
@@ -204,6 +204,12 @@
         "icon": "$(output)",
         "category": "Confluent"
       },
+      {
+        "command": "confluent.showSidecarOutputChannel",
+        "title": "Show Sidecar Output Channel",
```

Comment:
Do we expose "sidecar" as a term to the user? I don't remember.

Reply:
Very rarely, I think the most visible spot for extension users is in https://github.com/confluentinc/vscode?tab=readme-ov-file#outputs